src/test/java/com/basho/riak/client/core/util/DefaultCharsetTest.java

```diff
@@ -0,0 +1,49 @@
+package com.basho.riak.client.core.util;
+
+import java.nio.charset.Charset;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class DefaultCharsetTest
+{
```

Comment:
It might be a good idea to fix the behavior for the case when unavailable charset is requested